cmd/wallet-web/src/main.js

```diff
@@ -5,14 +5,14 @@ SPDX-License-Identifier: Apache-2.0
 */
 
 import { createApp } from 'vue';
-import store from './store';
-import App from './App.vue';
-import router from './router/index';
+import { mapActions, mapGetters } from 'vuex';
```

Comment:
There is technically no change to this file right , i noticed linter keep switching these import, but if we haven't touched the file, shall we still push it ??

Reply:
I've touched it myself, not linter